cypress/support/fragments/inventory/createPageTypeRequest.js

```diff
@@ -0,0 +1,247 @@
+import uuid from 'uuid';
+import { HTML, including } from '@interactors/html';
+import {
+  Button,
+  MultiColumnListCell,
+  Section,
+  TextField,
+  Modal,
+  Checkbox,
+  Heading,
+  Link,
+  Pane,
+  Accordion,
+  MultiColumnList,
+} from '../../../../interactors';
+import getRandomPostfix from '../../utils/stringTools';
+
+const actionsButton = Button('Actions');
+const saveAndCloseButton = Button({ id: 'clickable-save-request' });
+const newRequestButton = Button('New Request');
+const selectUserModal = Modal('Select User');
+const loanAndAvailabilitySection = Section({ id: 'acc06' });
+const inventorySearch = TextField({ id: 'input-inventory-search' });
+const searchButton = Button('Search');
+
+function deleteRequestApi(requestId) {
```

Comment:
please use the same function from cypress/support/fragments/requests/requests.js (we already have function like this)